src/controllers/EventsController.php

```diff
@@ -499,12 +499,12 @@ public function putAction($request, $db)
             }
             $cfp_start_date = $request->getParameter("cfp_start_date", false);
             if (false !== $cfp_start_date && strtotime($cfp_start_date)) {
-                $cfp_start_date = new DateTime(strtotime($cfp_start_date), $tz);
+                $cfp_start_date = new DateTime('@' . strtotime($cfp_start_date), $tz);
```

Comment:
Never assume that we knew what we were doing at any point in the past :)
On 22 Feb 2015 12:14, "Andreas Heigl" <notifications@github.com> wrote:

> In src/controllers/EventsController.php
> <https://github.com/joindin/joindin-api/pull/134#discussion_r25131502>:
>
> > @@ -499,12 +499,12 @@ public function putAction($request, $db)
> >              }
> >              $cfp_start_date = $request->getParameter("cfp_start_date", false);
> >              if (false !== $cfp_start_date && strtotime($cfp_start_date)) {
> > -                $cfp_start_date = new DateTime(strtotime($cfp_start_date), $tz);
> > +                $cfp_start_date = new DateTime('@' . strtotime($cfp_start_date), $tz);
>
> You are definitely right there. As far as I recall my intention was to
> keep the the original code as untouched as possible. I was wondering why
> that was done in that way but thought it had it's reason. Therefore I just
> kept the strange code and worked around it.
>
> But it seems that the strtotome wasn't necessary in the first place.
> Therefore I'll remove it.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/joindin/joindin-api/pull/134/files#r25131502>.
>